### PR TITLE
#162170685 Article should show read time

### PR DESCRIPTION
**Pull request by oluwajuwon**
#### What does this PR do?

This PR enables users see read time of an article

#### Description of Task to be completed?

 - Write test for appropriate component
 - Create read time component

#### How should this be manually tested?
* Clone the repo and `cd` into the folder
* Run `npm install` to install all the dependencies
* Run `npm run start:dev` to start the server
* In your browser visit `localhost:3000/articles/:articleSlug` (a slug of an existing article) or check all the articles via `localhost:3000/articles` the read time should be displayed there

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories? (If applicable)
[#162170685](https://www.pivotaltracker.com/story/show/162170685)

#### Screenshots (If applicable)
screen shot 2019-01-31 at 1 37 20 am
screen shot 2019-01-31 at 1 37 44 am

#### Questions:
N/A

### CODE REVIEW

File: src/components/presentation/ReadTime.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+const ReadTime = ({ article }) => (
+  <div>
+    {article && (
+      <div className="read-time">
+        <i className="little material-icons" role="presentation">schedule</i>
+        <p>
+          {' '}
+          {article.readtime}
+          {' '}
+          read
```
Comment (omokaro-faith) [inline src/components/presentation/ReadTime.jsx:12]: Do you really need to put empty strings inside curly braces here?

File: src/components/presentation/ReadTime.test.jsx [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import { render } from 'react-testing-library';
+import ReadTime from './ReadTime';
+
+test('it renders', () => {
+  render(<ReadTime />);
+});
```
Comment (omokaro-faith) [inline src/components/presentation/ReadTime.test.jsx:16]: LoL. There's no test assertion here.
Please add assertions.

**Review by omokaro-faith — CHANGES_REQUESTED** [review]

File: src/components/presentation/ReadTime.test.jsx (hunk 2 above)
Comment (akhilome) [reply src/components/presentation/ReadTime.test.jsx:16]: 😁

File: src/components/presentation/ArticleCard.jsx [hunk 3]
```diff
@@ -1,9 +1,10 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
 import PropTypes from 'prop-types';
+import ReadTime from './ReadTime';
 
 const ArticleCard = ({ article }) => {
-  const defaultImg = process.env.REACT_ARTICLE_CARD_DEFAULT_IMG;
+  const defaultImg = process.env.REACT_APP_ARTICLE_CARD_DEFAULT_IMG;
```
Comment (Johnsonojo) [inline src/components/presentation/ArticleCard.jsx:7]: Good job renaming the value of `defaultImg`

File: src/components/presentation/ReadTime.jsx (hunk 1 above)
Comment (oluwajuwon) [reply src/components/presentation/ReadTime.jsx:12]: I've actually found a better way to go about it, I'll make the change now

File: src/components/presentation/ReadTime.test.jsx (hunk 2 above)
Comment (oluwajuwon) [reply src/components/presentation/ReadTime.test.jsx:16]: I'll work on that right away

**Review by frostyblok — APPROVED** [review]

**Review by akhilome — APPROVED** [review]

**Review by luqmanoop — APPROVED** [review]

**Review by omokaro-faith — APPROVED** [review]

**Pull request merged**